drivers/hmis/spec/support/graphql_helpers.rb

```diff
@@ -109,8 +109,11 @@ def error_fields
     ERRORS
   end
 
+  # FIXME: this is not supported by graphql. We should find another way to test types, such as
+  # https://graphql-ruby.org/testing/helpers.html
   def to_gql_input_object(values, klass, current_user: nil)
```

Comment:
Good point, we only had one usage that wasn't really necessary, opened https://github.com/greenriver/hmis-warehouse/pull/4770